src/commands/SendCommand.ts

```diff
@@ -0,0 +1,91 @@
+import { Message, MessageEmbed, TextChannel, DMChannel, NewsChannel } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import PermissionRegistry from '../permissions/PermissionRegistry';
+import Command from './Command';
+
+export default class ShutdownCommand extends PrefixCommand {
```

Comment:
I'm not very experienced in TypeScript, but this class being called `ShutdownCommand` seems a bit odd, when the file is `SendCommand` and `ShutdownCommand` is a completely separate command.

Reply:
You are correct, this is not the first time I've done this 😆